Makefile

```diff
@@ -86,6 +86,11 @@ ifeq ($(ENABLE_KVM), 1)
 CARGO_OSDK_ARGS += --qemu-args="-accel kvm"
 endif
 
+ifdef PREBUILT_INITRAMFS
+INITRAMFS_IMAGE := $(shell pwd)/$(PREBUILT_INITRAMFS)
+CARGO_OSDK_ARGS += --initramfs="$(INITRAMFS_IMAGE)"
+endif
```

Comment:
The usage of `PREBUILT_INITRAMFS` is surprising to readers since it's an environment variable, but other variables are defined in this Makefile.

How about defining `PREBUILT_INITRAMFS` just in this file just like other variables (e.g., BENCHMARK)?

Reply:
If I got your point correctly, `PREBUILT_INITRAMFS` is the same as other variables. Or to put it another way, I initially added this variable with the usage `make PREBUILT_INITRAMFS=...`. Then, I found that adding `PREBUILT_INITRAMFS` to every `make` command is a bit redundant, and that the Makefile's variables can also be passed through environment variable, so I extracted it.

Reply:
Ok, thanks for your explanation! Then the current code is also acceptable.